### ISSUE TITLE
Gracefully shutdown the browser

### ISSUE THREAD

**Issue by ankur22** [grafana/xk6-browser#423]
It would seem that the context is being cancelled which is causing the browser processor to be killed instead of gracefully being shutdown. The plan is to follow the steps outline below to see if in majority of cases we can correctly shutdown the browser process and avoid logging errors on `cmd.wait()`.

Sounds good about the new issue. [The context is cancelled after each iteration](https://github.com/grafana/k6/blob/be4fcb8cb4e30eebaa40d2c356e0be5cc1b21d2c/js/runner.go#L746-L748), besides the `RunContext` being cancelled at shutdown. So we should be able to stop the process gracefully [here](https://github.com/grafana/xk6-browser/blob/eb136726e37282fffad0c642062a04a6680a6adc/chromium/browser_type.go#L163), and ensure it works when running with multiple VUs/iterations, even though we can't ensure that it will work when k6 is stopped abruptly or at test end because of #2432.

_Originally posted by @imiric in https://github.com/grafana/xk6-browser/pull/419#discussion_r908462923_

**Comment by inancgumus** [comment]
For reference: https://github.com/grafana/k6/issues/2432#issuecomment-1170890617

**Comment by imiric** [comment]
I think this is now fixed after #535 and can be closed, right? @grafana/k6browserdev
